x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -63,6 +63,60 @@ setup:
   - match: { "test-index.mappings.properties.sparse_field.model_settings.task_type": sparse_embedding }
   - length: { "test-index.mappings.properties.sparse_field": 3 }
 
+---
+"Add field_caps support for semantic_text field types with sparse embedding":
+
+  - do:
+      field_caps:
+        include_empty_fields: false
```

Comment:
Can you also add a test when `include_empty_fields` is true. The field should be returned even if it's not populated yet.